samples/ComputerVision/AreaOfInterest/Program.cs

```diff
@@ -73,10 +73,10 @@ private static async Task RoiFromStreamAsync(ComputerVisionClient computerVision
             {
                 AreaOfInterestResult analysis = await computerVision.GetAreaOfInterestInStreamAsync(imageStream);
                 Console.WriteLine(imagePath);
-                Displayareas(analysis);
+                DisplayAreas(analysis);
             }
         }
-        private static void Displayareas(AreaOfInterestResult analysis)
+        private static void DisplayAreas(AreaOfInterestResult analysis)
```

Comment:
DisplayAreaOfInterest, so it is more clear and also indicates that there is only 1 area, not several